modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-validator/src/main/java/com/liferay/dynamic/data/mapping/validator/internal/DDMFormValidatorImpl.java

```diff
@@ -289,10 +289,20 @@ protected void validateDDMFormFieldValidationExpression(
 		}
 		catch (DDMExpressionException ddmee) {
 			throw new MustSetValidValidationExpression(
-				ddmFormField.getName(), validationExpression);
+				ddmFormField.getName(),
+				getExpressionValue(validationExpression));
 		}
 	}
 
+	private String getExpressionValue(String validationExpression) {
```

Comment:
Hey @linolaoj , Please move this method to the `MustSetValidValidationExpression` exception class. Thx